torchvision/models/detection/generalized_rcnn.py

```diff
@@ -74,7 +76,9 @@ def forward(self, images, targets=None):
         losses.update(proposal_losses)
 
         if torch.jit.is_scripting():
-            warnings.warn("RCNN always returns a (Losses, Detections tuple in scripting)")
+            if not self._has_warned:
+                warnings.warn("RCNN always returns a (Losses, Detections tuple in scripting)")
```

Comment:
While you're here do you mind fixing the typo: `RCNN always returns a (Losses, Detections) tuple in scripting`

Reply:
Sure, I can fix this.

And yeah, I agree that we should not be raising a warning all the time like this.